src/export/ProfileExporter.ts

```diff
@@ -22,7 +22,7 @@ export class ProfileExporter extends StructureDefinitionExporter {
           const structDef = this.exportStructDef(profile, tank);
           structDefs.push(structDef);
         } catch (e) {
-          logger.error(e.message);
+          logger.error(e.message, e.sourceInfo);
```

Comment:
Another quick comment...is there a reason that we don't want source info in the extensions? It seems that we explicitly log them for profiles, but there's no support added for extensions. There's probably a reason I'm just not aware of, but I want to clarify that.

Reply:
Nope, just forgot to add it in. I'll be sure to include it in the next set of updates after resolving the merge conflicts.

Reply:
Ah, okay. Sounds good. Then I guess as you do that, just make sure to update the corresponding tests for extensions and their exporter as well. Thanks!